<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>raiting</class>
 <widget class="QDialog" name="raiting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Рейтинг сотрудников</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_criteriaOption">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Выберите должность</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_showReport">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сгенерировать отчет</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/generate.png</normaloff>:/img/img/generate.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>График</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/chart.png</normaloff>:/img/img/chart.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>В период от</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="dateEdit_endReport">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="displayFormat">
           <string>dd MMMM yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2014</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QDateEdit" name="dateEdit_startReport">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="displayFormat">
           <string>dd MMMM yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2014</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>до</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_doljnost"/>
      </item>
     </layout>
     <zorder>pushButton_showReport</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_2</zorder>
     <zorder>comboBox_doljnost</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">QWidget
{
	background-color: rgb(211, 193, 255);
}
QLabel
{
	background-color: transparent;
}</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>843</width>
           <height>252</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resourses.qrc"/>
 </resources>
 <connections/>
</ui>
